### PR TITLE
CMO Hardsuit Protection Buff

### PR DESCRIPTION
**Pull request by BramvanZijp**
## About the PR
The following changes have been made to the CMOs Hardsuit.
Blunt: 0% > 20%
Slash: 0% > 25%
Piercing: 0% > 10%
Heat: 0% > 35%
Radiation: 0% > 40%
Caustic: 90% > 100%
Cellular: 0% > 50%
Explosive: 0% > 30%

Walk Slowdown: 10% > 15%
Sprint Slowdown: 5% > 10%

## Why / Balance
Because there is no reason for a HARDsuit to not protect you at all unless its made out of literal fucking plastic. (It just being made out of cheap plastic is now my headcanon for why it currently offers literally no protection against anything other then caustic.)
Also, caustic immunity makes sense if CE can be radiation immune, and RD can be p much explosive immune.
Lastly, the cellular resist will be nice for slimes or if #32136 gets merged assuming its cellular damage is changed to be effected by armor, since then the CMO will have specialized gear to not take nearly as much cellular damage from an implanter failure.

## Media
Not Neccesary.

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

**Changelog**
- tweak: The Chief Medical Officers Hardsuit now slightly protects the wearer, alongside making them immune to caustic substances and reducing genetic damage from external sources, but makes the wearer slightly slower.

### CODE REVIEW

**Comment by slarticodefast** [comment]
I don't think the CMO should be balanced for combat nor is it his job. It is already bad enough when he is fighting nukies with his hypospray. The hardsuit is for surviving in case of spacing and to retrieve dead crewmembers from outside the station, similar to the paramedic voidsuit.
The RDs hardsuit has restistances because Artifacts and anomalies tend to blow up.
The CE needs the rad protection for the singularity.

**Comment by Aisu9** [comment]
the implanter draw rework is made to **discourage** security or really anyone in position to remove an implant randomly check as a mechanical change to reduce the number of metachecks.
Please keep in mind that during a time when cloning could be research security used to take the entire science department hostage and force cloning research.
What this PR will bring is only push security to assault CMOs if not stealing only the hardsuit only to implant check with less problems

**Comment by BramvanZijp** [comment]
> the implanter draw rework is made to **discourage** security or really anyone in position to remove an implant randomly check as a mechanical change to reduce the number of metachecks. Please keep in mind that during a time when cloning could be researched security used to take the entire science department hostage and force cloning research. What this PR will bring is only push security to assault CMOs if not stealing only the hardsuit only to implant check with less problems

The idea is for security to call over the CMO to do the implant check, and if they do that too often then cmo can go "Just do it yourself", and stealing the hardsuit is literally against spacelaw and thus for security against server rules

**Comment by BramvanZijp** [comment]
> I don't think the CMO should be balanced for combat nor is it his job. It is already bad enough when he is fighting nukies with his hypospray. The hardsuit is for surviving in case of spacing and to retrieve dead crewmembers from outside the station, similar to the paramedic voidsuit. The RDs hardsuit has restistances because Artifacts and anomalies tend to blow up. The CE needs the rad protection for the singularity.

I get it that it shouldnt be balanced for combat, but there should be SOME level of protection, just like how CE has 20% reduction to the 3 brutes even though their job doesnt need to take that damage outside combat, because its still a HARDsuit.

**Comment by BramvanZijp** [comment]
> The point of the CMO hardsuit is to be mobile and unprotective. The CMO is not a frontline combatant, and they'll usually only be wearing their suit in an emergency regardless. CMO does not need combat armor for any reason, if they say they do then they're not doing their job as CMO.
> 
> Also don't do rp changelogs.

Its still a hardsuit which it feels odd that it offers no protection except caustic, even though the paramed voidsuit DOES offer a few light protections.

**Comment by Aisu9** [comment]
> > the implanter draw rework is made to **discourage** security or really anyone in position to remove an implant randomly check as a mechanical change to reduce the number of metachecks. Please keep in mind that during a time when cloning could be researched security used to take the entire science department hostage and force cloning research. What this PR will bring is only push security to assault CMOs if not stealing only the hardsuit only to implant check with less problems
> 
> The idea is for security to call over the CMO to do the implant check, and if they do that too often then cmo can go "Just do it yourself", and stealing the hardsuit is literally against spacelaw and thus for security against server rules

removing syndicate implant is not CMO's job, and despite being again server rule i can guarantee you that will 100% happen some players just don't give a shit about department SoP and rules.(botanist are already not giving a shit about doing their job on salamander, it can only get worse)

**Comment by BramvanZijp** [comment]
> > > the implanter draw rework is made to **discourage** security or really anyone in position to remove an implant randomly check as a mechanical change to reduce the number of metachecks. Please keep in mind that during a time when cloning could be researched security used to take the entire science department hostage and force cloning research. What this PR will bring is only push security to assault CMOs if not stealing only the hardsuit only to implant check with less problems
> > 
> > 
> > The idea is for security to call over the CMO to do the implant check, and if they do that too often then cmo can go "Just do it yourself", and stealing the hardsuit is literally against spacelaw and thus for security against server rules
> 
> removing syndicate implant is not CMO's job, and despite being again server rule i can guarantee you that will 100% happen some players just don't give a shit about department SoP and rules.(botanist are already not giving a shit about doing their job on salamander, it can only get worse)

Implant removal IMO is a medical procedure, so the CMO being called over to do it if they have the time do just encourages that bit of realism/roleplay. And if the CMO is busy or sec wants it done right away they can do it themselves, just gotta be sure you are correct to avoid the genetic damage.

**Comment by BramvanZijp** [comment]
> The point of the CMO hardsuit is to be mobile and unprotective. The CMO is not a frontline combatant, and they'll usually only be wearing their suit in an emergency regardless. CMO does not need combat armor for any reason, if they say they do then they're not doing their job as CMO.
> 
> Also don't do rp changelogs.

Also, I as CMO have pulled a HoPs body out of their office before while that office was actively being bombed, so there are situations in which minor protection is valid.

**Comment by Aisu9** [comment]
> > > > the implanter draw rework is made to **discourage** security or really anyone in position to remove an implant randomly check as a mechanical change to reduce the number of metachecks. Please keep in mind that during a time when cloning could be researched security used to take the entire science department hostage and force cloning research. What this PR will bring is only push security to assault CMOs if not stealing only the hardsuit only to implant check with less problems
> > > 
> > > 
> > > The idea is for security to call over the CMO to do the implant check, and if they do that too often then cmo can go "Just do it yourself", and stealing the hardsuit is literally against spacelaw and thus for security against server rules
> > 
> > 
> > removing syndicate implant is not CMO's job, and despite being again server rule i can guarantee you that will 100% happen some players just don't give a shit about department SoP and rules.(botanist are already not giving a shit about doing their job on salamander, it can only get worse)
> 
> Implant removal IMO is a medical procedure, so the CMO being called over to do it if they have the time do just encourages that bit of realism/roleplay. And if the CMO is busy or sec wants it done right away they can do it themselves, just gotta be sure you are correct to avoid the genetic damage.

the whole point of the implant rework PR is to **MAKE SURE THEY STOP RANDOMLY CHECK PEOPLE**
What you are **providing is a way for them to CONTINUE random metachecks**

**Comment by Aisu9** [comment]
you are just giving ways to enable a behaviour that another PR aim to cull if not remove entirely

**Comment by BramvanZijp** [comment]
Then that PR can choose to do what it is doing right now and have said cellular damage ignore armor. Im just saying it could be interesting if the CMO in their hardsuit took less damage, but that would still require the implanter PR to be changed, which they can choose not to do.

**Comment by BramvanZijp** [comment]
The general point of this PR is to give the CMO's hardsuit low-level combat protection, which both CE and RD also do even for damage types not relevant to their jobs. As for the genetic resistance, Thats just generally a cool and unique feature for the CMO's hardsuit to have, and in the current state of the implanter rework PR, will have no effect on it, though I personally think it should.

**Comment by Everturning** [comment]
>what this PR will bring is only push security to assault CMOs if not stealing the hardsuit only to implant check with less problems

even if people would do this this is against the rules and so people wont. it's like if sec assaulted RD to steal their suit for defusing a bomb. they don't do that.

**Comment by superjj18** [comment]
> I don't think the CMO should be balanced for combat nor is it his job. It is already bad enough when he is fighting nukies with his hypospray. The hardsuit is for surviving in case of spacing and to retrieve dead crewmembers from outside the station, similar to the paramedic voidsuit.
> 
> The RDs hardsuit has restistances because Artifacts and anomalies tend to blow up.
> 
> The CE needs the rad protection for the singularity.

By this logic CE and RD should lose all other resistances since they make it, as you say, "balanced for combat".

Are you really saying that the CMO hardsuit offers the same level of protection as a T-shirt?(IE none)

**Comment by BramvanZijp** [comment]
> > I don't think the CMO should be balanced for combat nor is it his job. It is already bad enough when he is fighting nukies with his hypospray. The hardsuit is for surviving in case of spacing and to retrieve dead crewmembers from outside the station, similar to the paramedic voidsuit.
> > The RDs hardsuit has restistances because Artifacts and anomalies tend to blow up.
> > The CE needs the rad protection for the singularity.
> 
> By this logic CE and RD should lose all other resistances since they make it, as you say, "balanced for combat".
> 
> Are you really saying that the CMO hardsuit offers the same level of protection as a T-shirt?(IE none)

This is exactly my point.

**Comment by ScarKy0** [comment]
I am for this PR.
I didn't even know CMO suit grants no protections, which is kinda against what most hardsuits do?
And as said above, if CMO having low-level combat protection is bad it should also be removed from RD and CE suit. The only differance here is that hypospray is being used for validhunting, which should be fixed by itself and not used an excuse as to why PRs similiar to this one shouldn't go through.

**Comment by BramvanZijp** [comment]
> I am for this PR. I didn't even know CMO suit grants no protections, which is kinda against what most hardsuits do? And as said above, if CMO having low-level combat protection is bad it should also be removed from RD and CE suit. The only differance here is that hypospray is being used for validhunting, which should be fixed by itself and not used an excuse as to why PRs similiar to this one shouldn't go through.

What if the hypopen and cmo hypospray had a 0.5 or 1 second do-after bar if used on somebody else who is wearing a hardsuit?

**Comment by ScarKy0** [comment]
> What if the hypopen and cmo hypospray had a 0.5 or 1 second do-after bar if used on somebody else who is wearing a hardsuit?

Then that would be fine. All I'm saying is if the only arguement for CMO not having this sort of protection on their hardsuit is the fact they validhunt, this is the same as saying security guns are too strong so you nerf their armor to balance it out. It doesn't tackle the problem at the core.

**Comment by BramvanZijp** [comment]
If people think some of these protections should be tweaked, sure I can understand that, leave a comment and tell me what you think should change. But if you think the CMO hardsuit should stay as a defenseless hardsuit outside of job-related damages, I will be PRing the same for the CE and RD.

**Comment by superjj18** [comment]
> removing syndicate implant is not CMO's job, and despite being again server rule i can guarantee you that will 100% happen some players just don't give a shit about department SoP and rules.(botanist are already not giving a shit about doing their job on salamander, it can only get worse)

Then ahelp and get them banned/rolebanned so the community gets better?  Seccies and command are held to higher standards

**Comment by Aisu9** [comment]
> > what this PR will bring is only push security to assault CMOs if not stealing the hardsuit only to implant check with less problems
> 
> even if people would do this this is against the rules and so people wont. it's like if sec assaulted RD to steal their suit for defusing a bomb. they don't do that.

Security used to take science hostage and threaten to gib them if they were not researching cloning, i have yet to know if they were banned of just get a tap on the wrist for their actions

**Comment by BramvanZijp** [comment]
> > > what this PR will bring is only push security to assault CMOs if not stealing the hardsuit only to implant check with less problems
> > 
> > 
> > even if people would do this this is against the rules and so people wont. it's like if sec assaulted RD to steal their suit for defusing a bomb. they don't do that.
> 
> Security used to take science hostage and threaten to gib them if they were not researching cloning, i have yet to know if they were banned of just get a tap on the wrist for their actions

This feels like a drastic overstatement.  Besides, the implanter draw PR ignores armor resistances currently, so in its current state these PRs are not relevant to eachother.

**Comment by BramvanZijp** [comment]
So heres my question for everybody who disliked this PR:
Do you want these resistances added, or would you prefer we remove the irrelevant ones from CE and RD too? Because currently it just seems unfair towards the CMO.

**Comment by BramvanZijp** [comment]
Added a slowdown nerf to compensate for the extra protection balance wise.

**Comment by LankLTE** [comment]
Like others have said, the CMO's hardsuit is one of the fastest in the game for the tradeoff of being unprotective. I think this PR changing it to be a more traditional hardsuit misses the point entirely and is frankly going to make it worse.

**Comment by superjj18** [comment]
> Like others have said, the CMO's hardsuit is one of the fastest in the game for the tradeoff of being unprotective. I think this PR changing it to be a more traditional hardsuit misses the point entirely and is frankly going to make it worse. 

It's a hardsuit not a voidsuit, I agree the speed should be untouched, but there should be some low level basic protections.

**Comment by EthanQix** [comment]
From a design PoV, this PR is doing two things wrong : 
- Making CMO more combat viable is something we DO NOT want. This was the entire point of giving a doAfter to the hypospray : avoiding CMO running into melee range and injecting nukies with nasty juice. CMO is a non combat role and so their hardsuit should not provide any meaningful combat advantage.
- Negating genetic damage comes handy in only a couple of situations : fighting feral slimes, which CMO shouldn't be doing, and random implant checks, which we DO NOT want to happen in the first place.

Long story short : It's called a hardsuit because it protects the wearer from the vacuum of space, and CMO's hardsuit is pretty good at it given how small its speed debuff is. Giving it more stats makes negative sense.

**Comment by BramvanZijp** [comment]
> From a design PoV, this PR is doing two things wrong.
> 
> * Making CMO more combat viable is something we DO NOT want. This was the entire point of giving a doAfter to the hypospray : avoiding CMO running into melee range and injecting nukies with nasty juice. CMO is a non combat role and so their hardsuit should not provide any meaningful combat advantage.
> * Negating genetic damage comes handy in only a couple of situations : fighting feral slimes, which CMO shouldn't be doing, and random implant checks, which we DO NOT want to happen in the first place.
> 
> Long story short - it's called a hardsuit because it protects the wearer from the vacuum of space, and CMO's hardsuit is pretty good at it given how small its speed debuff is. Giving it more stats makes negative sense.

So should I remove the combat resistances from RD and CE aswell then since they also are non combat roles?

**Comment by EthanQix** [comment]
> > From a design PoV, this PR is doing two things wrong.
> > 
> > * Making CMO more combat viable is something we DO NOT want. This was the entire point of giving a doAfter to the hypospray : avoiding CMO running into melee range and injecting nukies with nasty juice. CMO is a non combat role and so their hardsuit should not provide any meaningful combat advantage.
> > * Negating genetic damage comes handy in only a couple of situations : fighting feral slimes, which CMO shouldn't be doing, and random implant checks, which we DO NOT want to happen in the first place.
> > 
> > Long story short - it's called a hardsuit because it protects the wearer from the vacuum of space, and CMO's hardsuit is pretty good at it given how small its speed debuff is. Giving it more stats makes negative sense.
> 
> So should I remove the combat resistances from RD and CE aswell then since they also are non combat roles?

Honestly ? If they're unlikely to meet a specific hazard in their line of work, their hardsuit should probably not have any significant protection against it.

**Comment by superjj18** [comment]
> From a design PoV, this PR is doing two things wrong.

> * Making CMO more combat viable is something we DO NOT want. This was the entire point of giving a doAfter to the hypospray : avoiding CMO running into melee range and injecting nukies with nasty juice. CMO is a non combat role and so their hardsuit should not provide any meaningful combat advantage.

>10% piercing damage reduction 

This change doesn’t stop CMO from getting obliterated in 99% of serious combat situations, so saying that this pr makes it “combat viable” is a hilarious and disingenuous overstatement.  At best it helps against med/evac bombings, NPC enemies, and unruly tiders. 

> * Negating genetic damage comes handy in only a couple of situations : fighting feral slimes, which CMO shouldn't be doing, and random implant checks, which we DO NOT want to happen in the first place.

Every single person on the station fights feral slimes, from HOS to engineers to greytiders, because they spawn anywhere and everywhere, including medbay.

also your point about implant checks is totally moot as it doesn’t even negate genetic damage 


> Long story short - it's called a hardsuit because it protects the wearer from the vacuum of space, and CMO's hardsuit is pretty good at it given how small its speed debuff is. Giving it more stats makes negative sense.

This is incorrect as well since the emergency space suits are not hard-suits and as such have no resistances or heat insulation.  It’s called a hardsuit because it’s HARD.

**Comment by BramvanZijp** [comment]
> > From a design PoV, this PR is doing two things wrong.
> 
> > * Making CMO more combat viable is something we DO NOT want. This was the entire point of giving a doAfter to the hypospray : avoiding CMO running into melee range and injecting nukies with nasty juice. CMO is a non combat role and so their hardsuit should not provide any meaningful combat advantage.
> 
> This change doesn’t stop CMO from getting obliterated in 99% of serious combat situations, so saying that this pr makes it “combat viable” is a hilarious and disingenuous overstatement. At best it helps against bombings, NPC enemies, and unruly tiders.
> 
> > * Negating genetic damage comes handy in only a couple of situations : fighting feral slimes, which CMO shouldn't be doing, and random implant checks, which we DO NOT want to happen in the first place.
> 
> Every single person on the station fights feral slimes, from HOS to engineers to greytiders, because they spawn anywhere and everywhere, including medbay.
> 
> also your point about implant checks is totally moot as it doesn’t even negate genetic damage
> 
> > Long story short - it's called a hardsuit because it protects the wearer from the vacuum of space, and CMO's hardsuit is pretty good at it given how small its speed debuff is. Giving it more stats makes negative sense.
> 
> This is incorrect as well since the emergency space suits are not hard-suits and as such have no resistances or heat insulation. It’s called a hardsuit because it’s HARD.

It negates genetic damage, just implant checks currently ignore armor.

**Comment by ps3moira** [comment]
the hardsuit should just be for biohazards and disease protection, there shouldn't be any actual protection for combat purposes.

**Comment by Kadeo64** [comment]
I agree with the caustic change and I can see the genetic/cellular damage change being alright if there were actual ways to deal that damage aside from metabolization but i don't think the hardsuit needs anything else.

**Comment by EthanQix** [comment]
> This change doesn’t stop CMO from getting obliterated in 99% of serious combat situations, so saying that this pr makes it “combat viable” is a hilarious and disingenuous overstatement. At best it helps against med/evac bombings, NPC enemies, and unruly tiders.

You will notice I said "*more* combat viable". The PR will obviously not make the CMO suit on par with stuff like sec suits. My point is, the CMO hardsuit is absolute garbage in combat by design. It should stay absolute garbage in combat.

> Every single person on the station fights feral slimes, from HOS to engineers to greytiders, because they spawn anywhere and everywhere, including medbay.

True, and when that happens, there's no reason for CMO to be better than the other doctors at fighting them.

> also your point about implant checks is totally moot as it doesn’t even negate genetic damage

I meant Cellular, my bad. Also the failed implant extraction damage bypasses armor, so my point was indeed moot.

> This is incorrect as well since the emergency space suits are not hard-suits and as such have no resistances or heat insulation. It’s called a hardsuit because it’s HARD.

Well semantics aside - any command suit shouldn't have resistances against hazards that are unlikely to be met during their job. And maybe non armored suits should not be named hardsuits anyway.

**Comment by thebadman4662** [comment]
> You will notice I said "more combat viable". The PR will obviously not make the CMO suit on par with stuff like sec suits. My point is, the CMO hardsuit is absolute garbage in combat by design. It should stay absolute garbage in combat.

You are medic, your job is to get in ASAP and get out ASAP if need be with crit/dead people so you can patch them up behind cover and fight in your stead. Nerfing CMO suit speed is nerfing their core values the way I see it.

**Comment by Spessmann** [comment]
Solution: keep resistances the same and rename to "CMO eva suit" :trollface:

**Comment by BolloTea** [comment]
Alternatively, you can give 5% resistance to physical and thermal damage, 20-25% explosion and 40-50 acid.

**Comment by thebadman4662** [comment]
> Alternatively, you can give 5% resistance to physical and thermal damage, 20-25% explosion and 40-50 acid.

Caustic IS the acid. Nevermind the fact HOW SHIT for the effort touch damage is on acids, even the best one of the 3 and that many civilians have 25%+ resistance to it because coats. Razorium at least gives bleeding on top.

**Comment by BolloTea** [comment]
> > Alternatively, you can give 5% resistance to physical and thermal damage, 20-25% explosion and 40-50 acid.
> 
> Caustic IS the acid. Nevermind the fact HOW SHIT for the effort touch damage is on acids, even the best one of the 3 and that many civilians have 25%+ resistance to it because coats. Razorium at least gives bleeding on top.

I forgot that the primary protection against acid is 90%, corrected the text

**Comment by SlamBamActionman** [comment]
Heya, the maintainer team has decided to close this PR for the following reasons:

- Moves CMO towards being more combat-able, which does not fit with its role.
- Medical (along with CMO) is directly beneficial to the station in a combat scenario when performing its main role, and are intended to be part of the backline; the hardsuit providing space protection with little slowdown is suitable for this.

The balance team are not opposed to making changes to hardsuits, but they should be suitable to the role that wears it. We recognize there are no design docs for Medical currently, however we feel the PR is not suitable for CMO and their intended gameplay. Wish the best with future contributions!
